### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by guern1kn**

### CODE REVIEW

**Review by afebbraro — APPROVED** [review]
good stuff! Approving but We'll want to merge right after release

**Comment by guern1kn** [comment]
Update the spark-angular package to the released stable version 12 before merging.

**Comment by guern1kn** [comment]
Make another PR to update the package versions on the main branch.

**Pull request merged**
